typing/jkind.ml

```diff
@@ -2272,31 +2272,36 @@ let sub_jkind_l ~type_equal ~jkind_of_type sub super =
   let bounds =
     List.for_all
       (fun (Axis.Pack axis) ->
-        let (module Axis_ops) = Axis.get axis in
+        let (module Bound_ops) = Axis.get axis in
         let bound1 = Bounds.get ~axis sub.jkind.upper_bounds in
         let bound2 = Bounds.get ~axis super.jkind.upper_bounds in
+        let with_bounds1 =
+          With_bounds.types_on_axis ~axis sub.jkind.with_bounds
```

Comment:
This does some work, and it will be wasted effort if the second shortcut below is taken.